lib/ansiblelint/generate_docs.py

```diff
@@ -4,9 +4,10 @@
 import importlib
 from inspect import getmembers, ismodule, isclass
 import logging
-import rules
+import rules  # type: ignore
```

Comment:
why do you need to ignore it?

Reply:
Yep, because it was an invalid import, correctly identified by mypy. Still, this will go away once we merge https://github.com/ansible/ansible-lint/pull/783